core/field_colour.js

```diff
@@ -314,26 +453,51 @@ Blockly.FieldColour.prototype.dropdownCreate_ = function() {
   // Create the palette.
   var table = document.createElement('table');
   table.className = 'blocklyColourTable';
+  table.tabIndex = "0";
```

Comment:
nit: Should be ' '

Reply:
actually should be a number haha, but thanks for spotting this.